odl/tomo/backends/astra_cuda.py

```diff
@@ -372,7 +372,8 @@ def astra_cuda_bp_scaling_factor(proj_space, reco_space, geometry):
         if isinstance(geometry, Parallel2dGeometry):
             # Scales with 1 / cell_volume
             scaling_factor *= float(reco_space.cell_volume)
-        elif isinstance(geometry, FanFlatGeometry):
+        elif isinstance(geometry, FanBeamGeometry)\
+                and geometry.det_curve_radius is None:
```

Comment:
Last little thing: Please use parentheses instead of backslash to continue the line.